### PR TITLE
Move server admin to oc

### PR DESCRIPTION
**Pull request by damemi**
This rolls up another useless helper package

### CODE REVIEW

File: pkg/oc/cli/admin/cert/cert.go [hunk 1]
```diff
@@ -6,7 +6,7 @@ import (
 	"k8s.io/cli-runtime/pkg/genericclioptions"
 	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
 
-	"github.com/openshift/origin/pkg/cmd/server/admin"
+	"github.com/openshift/origin/pkg/oc/cli/admin/server"
```
Comment (deads2k) [inline pkg/oc/cli/admin/cert/cert.go]: I'm sure you can see the pattern in the packages.  Match it please.

File: test/integration/front_proxy_test.go [hunk 2]
```diff
@@ -23,7 +23,7 @@ import (
 	"github.com/openshift/origin/pkg/api/legacy"
 	authorizationapi "github.com/openshift/origin/pkg/authorization/apis/authorization"
 	authorizationclient "github.com/openshift/origin/pkg/authorization/generated/internalclientset"
-	"github.com/openshift/origin/pkg/cmd/server/admin"
+	"github.com/openshift/origin/pkg/oc/cli/admin/server"
```
Comment (deads2k) [inline test/integration/front_proxy_test.go]: how does this compile?

**Comment by deads2k** [comment]
squash your test changes in.  Fix it to work. then lgtm

File: test/integration/front_proxy_test.go (hunk 2 above)
Comment (damemi) [reply test/integration/front_proxy_test.go]: oh, because I didn't try `make test` just `make`. updated

**Comment by damemi** [comment]
covered by https://github.com/openshift/origin/pull/23059

**Pull request closed without merging**